security/README.md

```diff
@@ -16,6 +16,14 @@ deployed services at risk of compromise or loss of privacy.
 The following guidelines apply to how you as an individual secure access to your
 systems (laptop, accounts, etc.) and communication (email, etc.).
 
+## Protecting Personal or Identifying Information
+
+Some individuals, due to their backgrounds or circumstances, might be at a
+higher risk of harmful consequences from privacy violations.
```

Comment:
I'm very open to feedback on this. I didn't know if it was helpful to give specifc examples or not.

Reply:
I think specific examples would definitely be helpful! It would help people understand the particular needs of those users. Here are a few we could include:

- **➡️ Survivors of domestic abuse ⬅️** or those who are trying to escape domestic abuse
- LGBTQIA+ individuals, especially if their housing is dependent on someone who might evict them upon learning this, or if they're in a number of other unsafe circumstances
- Political dissidents, asylum seekers, targets of government-sanctioned violence, etc (I feel pretty iffy on whether these are the right terms for what I'm thinking of; someone please suggest adjustments if they come to mind)
- Undocumented immigrants?

I think we could also include something about the necessity of being able to block other users or make their content only visible to specific users (_i.e._ denylist or allowlist).

Reply:
> I think we could also include something about the necessity of being able to block other users or make their content only visible to specific users (i.e. denylist or allowlist).

I'd be quite interested in a full document in this directory (or another?) that focuses on how to design a product such that users will feel safe. Moderation, name changing, SEO privacy, etc.

Reply:
> I'd be quite interested in a full document in this directory (or another?) that focuses on how to design a product such that users will feel safe. Moderation, name changing, SEO priva

I agree. I think this warrants a separate guide. In the meantime, do we think this current iteration is valuable as is, so long as we provide a few examples?

The ones @thoughtbot-summer posted are an excellent start.

Reply:
@thoughtbot-summer @mike-burns I want ahead and created a separate section for this, and included the examples.

I think there were some open questions on the language used in the examples, and I am open to suggestions 👍 

dc1c95e4132ab454d0a9299653ebbfe65069ecfa

Reply:
Thanks for that, I think this is a better organization structure.